pages/blog/hacktoberfest-summary-2020.md

```diff
@@ -0,0 +1,430 @@
+
+It doesn't matter. 
+
+Please don't make assumptions; assumptions are evil. Be opened, treat every contribution equally, and remember that the onboarding process is a crucial element. If you fail with the onboarding process, you fail big time at the very beginning.
+
+### What We Prepared
+
+We prepared the following materials for potential participants:
+- [Blog post](https://www.asyncapi.com/blog/hacktoberfest-2020) about our participation
+- [Onboarding videos](https://www.youtube.com/playlist?list=PLbi1gRlP7pigFSE_6G88x0t12HFLU4A4b) that explain how to start
+- 78 issues from more than 30 repositories and put them all in one [list](https://docs.google.com/spreadsheets/d/1vX4J395apexutfQ0OSqPNltFKDacmemHZwCmOXwHNLo/) with additional information about the difficulty level or the technical area,
+
+It took me around eight workdays to do it all. The most time-consuming part was to go through all the issues, pick candidates, create new ones, and group them all in a Google Sheet.
+
+### What We Did During The Event
+
+70 pull requests mean 70 reviews :sweat:
+
+If you know your project well, it is not very time-consuming, and anyway, it is the work you have to do as a maintainer. I do not count this time as an extra Hacktoberfest effort. Of course, it can be overwhelming, if this is not a standard amount of PRs that you get every month.
+
+We also hosted [office hours](https://www.youtube.com/playlist?list=PLbi1gRlP7pigZP2da6zbDT2YU7glQOTPH) for participants. This was fun, and we wanted to start doing live streams about AsyncAPI anyway.
+
+Last but not least, once a week, I advertised our project on the official Hacktoberfest Discord server.
+
+[![](/img/posts/hacktoberfest-summary-2020/feedback-sharkham.webp)](https://github.com/asyncapi/vscode-extension/pull/8#issuecomment-719759737)
+
+It looks like we do not need to do more for the next year.
+
+## Was It Worth It
+
+Hell yeah, and I'm already looking forward to Hacktoberfest 2021.
+
+It was great to see so many different people interacting with the project and seeing we reached our goal.
+
+[![](/img/posts/hacktoberfest-summary-2020/feedback-juergenbr.webp)](https://github.com/asyncapi/html-template/pull/98#issuecomment-709014754)
+
+We got some new features, CI/CD cleanup in all repositories, and solved many trivial SonarCloud-reported issues that we would never found time to solve.
+
+## What Made It Such a Success
+
+Our success was a typical return on investment.
+
+[![](/img/posts/hacktoberfest-summary-2020/feedback-Orodan.webp)](https://github.com/asyncapi/playground/pull/38#issuecomment-710014222)
+
+We asked all contributors to provide feedback on how they learned about us and what was the most helpful resource. 20 out of 26 responded to our request.
+
+![](/img/posts/hacktoberfest-summary-2020/how-they-learned.webp)
+_Figure 1: Ways in which the contributors learned about AsyncAPI participating in Hacktoberfest_
```

Comment:
You may want to use the [`<Figure>` component](https://github.com/asyncapi/website/blob/master/components/Figure.js):

```suggestion
<Figure
  src="/img/posts/hacktoberfest-summary-2020/how-they-learned.webp"
  caption="Figure 1: Ways in which the contributors learned about AsyncAPI participating in Hacktoberfest"
/>
```